app/scripts/controllers/dashboard_controller.js

```diff
@@ -0,0 +1,13 @@
+import App from 'scripts/application';
+import DashboardView from 'scripts/views/dashboard/dashboard';
+import DashboardArticles from 'scripts/collections/dashboard_articles';
+
+export default class DashboardController extends Marionette.Controller {
+  index() {
+    let articles = new DashboardArticles();
+
+    articles.fetch().then(function() {
```

Comment:
In my opinion it's not correct b/c `=>` function changes context. In this example it doesn't break anything b/c there is no `this` inside the function but it still can confuse.